Automattic/simplenote-ios
# Charlie/add sustainer icon

### Fix
As a thank you to our sustainers we are offering them the option to set their app icon to an exclusive icon color.  This PR adds that functionality.  NOTE: this will only work with existing sustainers, and becoming a sustainer is removed in another PR.  In a future PR will figure out how sustainers will continue having access to this option after sustainer is retired.

| Settings | Icon |
|--------|--------|
| <img src="https://cdn-std.droplr.net/files/acc_593859/z2xYls" /> | <img src="https://cdn-std.droplr.net/files/acc_593859/3xoghA" /> | 

### Test
1. Clean install of Simplenote.  Log into a non sustainer account.  Go to Tags > Settings and confirm that there is no row for enabling the sustainer app icon
2. Go to `SPSettingsViewController.swift. and add `return true` to L.44 so that you appear to be a sustainer
3. relaunch the app, return to the settings view and confirm that you have a switch under the Sustainer Thank you section
4. Enable the switch.  Then exit the app and confirm the icon is now the pretty green color
5. Relaunch the app, return to the settings and confirm that the sustainer app icon is enabled.

### Review
***(Required)*** Add instructions for reviewers.  For example:
> Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
***(Required)*** Add a concise statement to `RELEASE-NOTES.txt` if the changes should be included in release notes. Include details about updating the notes in this section. For example:
> `RELEASE-NOTES.txt` was updated in d3adb3ef with:
> 
> > Added markdown support

If the changes should not be included in release notes, add a statement to this section. For example:

> These changes do not require release notes.
